managers/tests/test_views.py

```diff
@@ -225,3 +229,89 @@ def test_delete_project_function_view_should_not_delete_project_on_non_admin_req
         response = self.client.post(self.url)
         self.assertEqual(response.status_code, 302)
         self.assertEqual(Project.objects.all().count(), 1)
+
+
+class ManageTaskActivitiesInProjectView(TestCase):
+    def setUp(self):
+        super().setUp()
+        self.user_admin = AdminUserFactory()
+        self.client.force_login(self.user_admin)
+        self.default1_task_activity = TaskActivityTypeFactory(is_default=True)
```

Comment:
We use snake case style of naming variables so instead of `default1....` please change it to `default_task_activity_1`